build.sbt

```diff
@@ -851,3 +850,154 @@ commands ++= {
+      moduleDeps("compiler").value,
+      // moduleDeps("dist").value,                // No sources, therefore no module in IntelliJ
+      moduleDeps("forkjoin").value,
+      moduleDeps("interactive").value,
+      moduleDeps("junit").value,
+      moduleDeps("library").value,
+      // moduleDeps("library-all").value,         // No sources
+      moduleDeps("manual").value,
+      moduleDeps("partest-extras").value,
+      moduleDeps("partest-javaagent").value,
+      moduleDeps("reflect").value,
+      moduleDeps("repl").value,
+      moduleDeps("repl-jline").value,
+      // moduleDeps("repl-jline-embedded").value, // No sources
+      // moduleDeps("root").value,                // No sources
+      // moduleDeps("scala-dist").value,          // No sources
+      moduleDeps("scaladoc").value,
+      moduleDeps("scalap").value,
+      moduleDeps("test").value)
+  }
+
+  def moduleDep(name: String, jars: Seq[File]) = {
+    val entries = jars.map(f => s"""        <root url="jar://${f.getCanonicalPath}!/" />""").mkString("\n")
+    s"""|    <library name="${name}-deps">
+        |      <CLASSES>
+        |$entries
+        |      </CLASSES>
+        |      <JAVADOC />
+        |      <SOURCES />
+        |    </library>""".stripMargin
+  }
+
+  def starrDep(jars: Seq[File]) = {
+    val entries = jars.map(f => s"""          <root url="file://${f.getCanonicalPath}" />""").mkString("\n")
+    s"""|    <library name="starr" type="Scala">
+        |      <properties>
+        |        <option name="languageLevel" value="Scala_2_12" />
+        |        <compiler-classpath>
+        |$entries
+        |        </compiler-classpath>
+        |      </properties>
+        |      <CLASSES />
+        |      <JAVADOC />
+        |      <SOURCES />
+        |    </library>""".stripMargin
+  }
+
+  def replaceLibrary(data: Node, libName: String, libType: Option[String], newContent: String) = {
+    object rule extends RewriteRule {
+      var transformed = false
+      def checkAttrs(attrs: MetaData) = {
+        def check(key: String, expected: String) = {
+          val a = attrs(key)
+          a != null && a.text == expected
+        }
+        check("name", libName) && libType.forall(tp => check("type", tp))
+      }
+
+      override def transform(n: Node): Seq[Node] = n match {
+        case e @ Elem(_, "library", attrs, _, _, _*) if checkAttrs(attrs) =>
+          transformed = true
+          XML.loadString(newContent)
+        case other =>
+          other
+      }
+    }
+    object trans extends RuleTransformer(rule)
+    val r = trans(data)
+    if (!rule.transformed) error(s"Replacing library classpath for $libName failed, no existing library found.")
+    r
+  }
+
+  val ipr = (baseDirectory in ThisBuild).value / "src/intellij/scala.ipr"
+
+  var continue = false
+  if (!ipr.exists) {
+    scala.Console.print(s"Could not find src/intellij/scala.ipr. Create new project files from src/intellij/*.SAMPLE (y/n)? ")
```

Comment:
Since it checks for `y` I'd call it `(y/N)` in the question to indicate that n is the default.